index.html

```diff
@@ -0,0 +1,136 @@
+                </ul>
+        </section>
+        <section id = "javascript_and_java"class = "main-section">
+            <header >
+                <h1>javascript and java</h1>
+               </header>
+               <p>avaScript and Java are similar in some ways but fundamentally different in some others.</p>
+               <p>The JavaScript language resembles Java but does not have Java's static typing and strong type checking</p>
+               <code>
+                num1 = 14;
+                num2 = 3;
+                mux = num1 *num2;
+                </code> 
+                <ul>
+                    <li>intro1</li>
+                    <li>intro2</li>
+                </ul>
+        </section>
+        <section id = "Hello_World"class = "main-section">
+            <header >
+                <h1>Hello World</h1>
+               </header>
+               <p> JavaScript follows most Java expression syntax, naming conventions and basic control-flow constructs which was the reason why it was renamed</p>
+               <p>In contrast to Java's compile-time system of classes built by declarations, JavaScript supports a runtime system based on a small number of data types </p>
+
+        </section>
+        <section id = "variables"class = "main-section">
+            <header >
+                <h2>variables</h2>
+               </header>
+               <p> JavaScript follows most Java expression syntax, naming conventions and basic control-flow constructs which was the reason why it was renamed</p>
+               <p>In contrast to Java's compile-time system of classes built by declarations, JavaScript supports a runtime system based on a small number of data types </p>
+               <code>
+                var n = 5;
+                var n1 = 6;
+                var divide = n / n1;
+                </code>
+                <ul>
+                    <li>variable dec1/li>
+                    <li>variable dec2</li>
+                </ul> 
+
+               
+        </section>
+        <section id = "declaring_variables"class = "main-section">
+            <header >
+                <h2>declaring variables</h2>
+               </header>
+               <p> JavaScript follows most Java expression syntax, naming conventions and basic control-flow constructs which was the reason why it was renamed</p>
+               <p>In contrast to Java's compile-time system of classes built by declarations, JavaScript supports a runtime system based on a small number of data types </p>
+               <code>
+                var x = 15;
+                var y = 6;
+                var z = x + y;
+                </code> 
+                <ul>
+                    <li>declarations1</li>
+                    <li>declarations2</li>
+                </ul>
+        </section>
+        <section id = "variable_scope"class = "main-section">
+            <header >
+                <h2>variable scope</h2>
+               </header>
```

Comment:
it could be better if you remove header tag hear. it will save your code some lines and make your code even cleaner

Reply:
- [ ] I  suggest you use back strings for areas where you indicate tag, for example, express the "header" tag as `header`.
- [ ] I strongly recommend you crosscheck your text before sending them. Grammarly does a good job with that to help you stay professional. from your review, you started Your sentences with small letters. 😃 . Kindly review this and make the required changes 👏